app/both/collections/residents.js

```diff
@@ -11,59 +11,58 @@ var ResidentsSchema = new SimpleSchema({
     type: String,
     autoform: {
       readonly() {
-        return !isUserAdmin() ? !isEdit(this) : false
-      }
+        return !isUserAdmin() ? !isEdit(this) : false;
+      },
     },
     custom() {
       return !isUserAdmin()
         ? isEdit(this)
-          ? "notAllowed"
+          ? 'notAllowed'
```

Comment:
This is somewhat dense. Let's make it a bit more verbose to improve readability.